mlora/config/lr_scheduler.py

```diff
@@ -14,6 +14,7 @@ class LRSchedulerConfig(DictConfig):
     def __init__(self, config: Dict[str, str]) -> None:
         super().__init__(config)
         self.init(self.__params_map, config)
+        self.last_epoch = -1
```

Comment:
why we need add new item named last_epoch?

Reply:
because the epoch is the only parameter to decide learning rate in function  torch.optim.lr_scheduler.CosineAnnealingLR which we use ,so we need this new item to make sure the learning rate to be same as the state we saved before.